packages/eslint-plugin-ui5-define-jsdocs/README.md

```diff
@@ -0,0 +1,43 @@
+# @ui5/eslint-plugin-ui5-define-jsdocs
```

Comment:
Align to readme format: see https://github.com/SAP/ui5-typescript/blob/master/packages/ts-types/README.md

Reply:
Done